client/src/components/QandA/QuestionList.jsx

```diff
@@ -2,18 +2,26 @@ import React from 'react';
 import Question from './Question';
 
 function QuestionList({
-  product, questions, questionHandler,
+  isLoaded, questions, product, questionHandler,
 }) {
-  console.log(questions);
   return (
-    <div className="questions">
-      {questions.map((question) => (
-        <Question
-          product={product}
-          questionHandler={questionHandler}
-          question={question}
-        />
-      ))}
+    <div>
+      {isLoaded
+        ? (
+          <p>Loading...</p>
```

Comment:
Good use of using conditional rendering to see if data is loaded yet and rendering 'Loading...' when it is not. I need to utilize this